src/core/PaperScope.js

```diff
@@ -105,6 +105,13 @@ var PaperScope = Base.extend(/** @lends PaperScope# */{
                 delete agent.webkit;
             if (agent.atom)
                 delete agent.chrome;
+            // In node context, JSDOM user agent no longer include "node" word,
+            // it is now called "jsdom". So we need to replace it because
+            // `agent.node` is expected to be true in node context.
+            if (agent.jsdom) {
+                agent.node = true;
+                delete agent.jsdom;
```

Comment:
@sasensi could you remove the `delete`? we can keep this.

Reply:
Even simpler and shorter:
`agent.node = !!agent.jsdom`

Reply:
ok